src/helpers/showFilterUtils.ts

```diff
@@ -25,7 +25,7 @@ const filterShowsByGenre = (showData: ShowData[], genreId: number): ShowData[] =
 const filterShowsByType = (showData: ShowData[], showType: 'movie' | 'tv'): ShowData[] => {
```

Comment:
@Thenlie Should we change this parameter to `media_type` to match the new naming convention?

Reply:
I think using camelCase for function params is still best. Just using snake case for json responses from TMDB.

Reply:
That's what I meant, `mediaType` or something of that nature. Just curious

Reply:
Yeah that is definitely a reasonable idea to keep the naming more consistent.